recipes/xtensor/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "0.24.0":
```

Comment:
While working on this recipe, can you also have a glance at the test package?
It's still using `tools.cross_building(self.settings)`.

Reply:
Oh..., I just bump up only.
I will fix it.

Reply:
fixed.